scijava-ops-engine/src/main/java/org/scijava/ops/engine/yaml/impl/JavaMethodYAMLInfoCreator.java

```diff
@@ -118,10 +118,15 @@ private Class<?> deriveOpType(String identifier, String typeString,
 					" could not be loaded: Inplaces must declare the index of the mutable parameter. For example, if your Inplace is designed to mutate the first argument, please write \"Inplace1\"");
 			}
 		}
-		else if (Pattern.matches("^[Cc]omputer\\s*[0-9]*$", typeString)) {
+		else if (Pattern.matches("^[Cc]omputer$", typeString)) {
 			return Computers.computerOfArity(parameterCount - 1);
 		}
-		else if (Pattern.matches("^[Ff]unction\\s*[0-9]*$", typeString)) {
+		else if (Pattern.matches("^[Cc]omputer\\s*[0-9]*$", typeString)) {
+			Integer a = Integer.parseInt(typeString.substring(typeString.indexOf(
```

Comment:
Why is this typed on `Integer` rather than `int`?

Reply:
`typeString.indexOf('r')` is always going to return 7. Why not just write `8` here? If the concern is with magic constants, then `'r'` is a magic constant here too; if we renamed `Computer` to `Crouton` it would also break the logic here, so robustness isn't really a reason to leave it as is.

Reply:
Also, this logic is buggy, because the string `Computer   9` will match, but throw `NumberFormatException` due to the leading spaces. You could put parentheses around the `[0-9]*` to capture it into a group, and then parse the group as a number.

Reply:
That logic should be fixed now!